### PR TITLE
Allow Access logs to be optional

### PR DESCRIPTION
**Pull request by malikvivek**
Allow output to file, stdout, stderr
Add fileCount parameter

Issue: #190

### CODE REVIEW

**Comment by rikesh-chouhan** [comment]
+1.
I'm wondering why the access Log Level is not configurable? Any reason?

**Comment by malikvivek** [comment]
I tried adding level. It picks it as a String and I couldn't figure out it's conversion to level type. Then post discussion with Mahesh, We decided to keep it at INFO and change when required from the finagle console.

**Comment by rikesh-chouhan** [comment]
+2

File: example/src/main/scala/com/lookout/borderpatrol/example/ServerConfig.scala [hunk 1]
```diff
@@ -11,6 +11,7 @@ import com.twitter.app.App
 import cats.data.Xor
 import com.twitter.finagle.http.path.Path
 import com.twitter.logging.Logger
+import com.twitter.logging.config.Level
```
Comment (maheshkelkar) [inline example/src/main/scala/com/lookout/borderpatrol/example/ServerConfig.scala]: we are not using this anymore

File: example/src/main/scala/com/lookout/borderpatrol/example/service.scala [hunk 2]
```diff
@@ -72,6 +71,13 @@ object service {
     "tokenmaster.oauth2" -> tokenmasterAccessIssuerChain(sessionStore)
   )
 
+
+  def accessLogFilter(config: ServerConfig): Filter[Request, Response, Request, Response] =
+    config.accessLogConfig match {
+    case Some(conf) => AccessLogFilter(conf.output, conf.fileSizeInMegaBytes, Level.INFO, conf.fileCount)
```
Comment (maheshkelkar) [inline example/src/main/scala/com/lookout/borderpatrol/example/service.scala]: remove Level.INFO

File: server/src/main/scala/com/lookout/borderpatrol/server/AccessLogFilter.scala [hunk 3]
```diff
@@ -40,10 +39,11 @@ case class AccessLogFilter(output: String, fileSizeInMegaBytes: Long, accessLogL
   val logger: Logger = {
     tap(Logger.get(loggerName)) { l =>
       l.clearHandlers()
-      // No need to set log level here so as to prevent the logger from forcing a log level
       // This allows the AccessLog logger to be used as a separate logger, not under root logger.
```
Comment (maheshkelkar) [inline server/src/main/scala/com/lookout/borderpatrol/server/AccessLogFilter.scala]: Actually it should read:
This allows the AccessLog logger to use different handlers

**Pull request merged**